File: lógica/Ex/cadastro.py
dados = {}
info = ['Nome', 'Endereço', 'Cidade', 'UF', 'CEP', 'Telefone', 'CPF',
        'RG', 'Data De Nascimento', 'Grau De Escolaridade', 'Curso']
for c in info:
    while True:
        dados[c] = str(input(f'{c}: ')).strip().title()
        if dados[c] != '':
            break
        else:
            print('Erro, insira os dados corretamente.')

for c in info:
    print(f'{c}: {dados[c]}')

    dados["Nome"] = "sim"
    print(dados["Nome"][2])

File: HTML e CSS/java/cadastro.js
var dados = {};
var info = ["Nome", "Endereço", "Cidade", "UF", "CEP", "Telefone", "CEP",
            "RG", "Data de Nascimento", "Grau de Escolaridade", "Curso"];

for(c of info){ 
    while (true){
        dados.c = prompt(c + ":")
        if (dados.c != ""){
            break
        }
        else{
            alert("Erro, insira os dados corretamente.")
        };
    };
};
for (c of info){
    alert(c + ": " + dados.c)
};


var objeto = {cachorro: ["pinscher", "bulldog"]}
console.log(objeto.cachorro[1])